OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -3,6 +3,18 @@
 import Foundation
 
 public struct OptimoveConfig {
+    /// Optimove initialization strategy.
+    struct InitializationStrategy: OptionSet {
```

Comment:
strategy is different from the android one

1. in android the goal of it is to tell which credentials will be eventually given. If it's a T&T client we dont need to initialize optimobile ever -- it would just needlessly keep caching events and initing other irrelevant stuff. So, we given it optimove_only meaning that 'later we will give only optimove creds and now we dont want optimobile init'
2. in ios it's currently not used, .delayed is not telling which creds we are planning to receive, optimobile is always initialised